frontend/packages/integration-tests-cypress/cy-command-additions/selectors.ts

```diff
@@ -0,0 +1,20 @@
+import { timeouts } from '../utils/timeouts';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      getElemByDataId(selector: string, timeout?: number): Chainable<Element>;
+      getElemByActionId(selector: string): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add('getElemByDataId', (selector, timeout = timeouts.default) =>
```

Comment:
Please follow console variable case conventions. Missing TypeScript types as well.

```suggestion
Cypress.Commands.add('getElemByDataID', (selector, timeout = timeouts.default) =>
```

Reply:
fixed